priv/repo/migrations/20240227102315_add_api_key_settings_to_settings_schema.exs

```diff
@@ -0,0 +1,69 @@
+defmodule Trento.Repo.Migrations.AddApiKeySettingsToSettingsSchema do
+  use Ecto.Migration
+
+  def change do
```

Comment:
omg, for my sanity I'm skipping this file review, and I trust it works. I don't understand half the things XD